From ce335250360779414304856efa3488cb30393109 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 6 Nov 2013 12:36:04 +0100
Subject: monitor: Remove host_net_add/remove for Red Hat Enterprise Linux

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <b28f4458b4470c9a6fcbf5085c3a1bb4d7a0a5eb.1383741033.git.mrezanin@redhat.com>
Patchwork-id: 55519
O-Subject: [RHEL7 qemu-kvm PATCH v2 4/4] monitor: Remove host_net_add/remove for Red Hat Enterprise Linux
Bugzilla: 1010858
RH-Acked-by: Michal Novotny <minovotn@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

This is forwardport of RHEL-6 commit dd94505bd1b826db0c7e155ccee5c24f77987f16:

    monitor: Remove host_net_add/remove for Red Hat Enterprise Linux

    The host_net_add/remove commands are replaced by netdev_add/del. Remove
    them.

    Signed-off-by: Amit Shah <amit.shah@redhat.com>

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
(cherry picked from commit c9963b7cbcc5a0148ce8c29dfa4c473b27f2fd71)
---
 hmp-commands.hx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index 91477c5..3cd63be 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -1338,6 +1338,7 @@ STEXI
 Inject PCIe AER error
 ETEXI
 
+#if 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "host_net_add",
         .args_type  = "device:s,opts:s?",
@@ -1367,6 +1368,7 @@ STEXI
 @findex host_net_remove
 Remove host VLAN client. Deprecated, please use @code{netdev_del} instead.
 ETEXI
+#endif
 
     {
         .name       = "netdev_add",
-- 
1.8.3.1

